Resources/ServerInfo/Guidebook/ServerRules/MoffRules/MoffR0.xml

```diff
@@ -0,0 +1,10 @@
+﻿<Document>
+  ## Play your character like a real person
+
+  This is an MRP/HRP server and you are expected to roleplay your character throughout the round.
+
+  Metagaming, Powergaming, etc. are not allowed.
+
+  You are expected to play to have fun, not play to "win", This goes for both security and antags.
```

Comment:
```suggestion
  You are expected to play to have fun, not play "to win", This applies to everyone, including both security and antagonists.
```

- `to win` something something split infinitive
- clarifying that it applies to "everyone" in case somebody wanted to rules lawyer and say "i'm a tider not sec or antag, this rule doesn't apply to me"
- "to go for" -> "to apply to" for the level of formality befitting Da Rulez